src/pages/Home/Home.jsx

```diff
@@ -11,23 +11,25 @@ import DoctorCarousel from '../../components/DoctorCarousel/DoctorCarousel'
 import OurPartners from '../../components/OurPartners/OurPartners'
 import LatestNews from '../../components/LatestNews/LatestNews'
 import Faq from '../../components/FAQ/Faq'
+import FirstViewUnlogged from '../../components/UnloggedMainView/FirstViewUnlogged'
 // faq modification
 
 const Home = () => {
+  const ROLE = localStorage.getItem('role')
```

Comment:
why is this info in localStorage? This is a security flaw.

Reply:
it will be updated in a new PR